pkgs/tools/package-management/nix/default.nix

```diff
@@ -87,14 +87,15 @@ in lib.makeExtensible (self: {
 
   stable = self.nix_2_8;
 
+  # remember to backport updates to the stable branch!
```

Comment:
22.05 right now is a special occasion but I don't think this should be general true cause updates can have breaking changes and be buggy.

Reply:
For unstable it might make sense?

Reply:
yeah for the unstable its ok